packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanel.tsx

```diff
@@ -861,8 +873,12 @@ export function TransferPanel() {
 
       // transaction submitted callback
       onTxSubmit(transfer)
-    } catch (ex) {
-      Sentry.captureException(ex)
+    } catch (error) {
+      Sentry.configureScope(function (scope) {
+        // tags only allow primitive values
+        scope.setTag('origin function', 'bridgeTransferStarter.transfer')
```

Comment:
Nice! Could we also add a bit more context here? Like which type of transfer and which token, if any?

Reply:
we do get the source & destination from the query params logged on Sentry already

as for the details we can always decode the data logged below by `error` so I am only adding verbose tags to the obviously high error rate sentry errors for now